### ISSUE TITLE
WP6: ISC Overlay breaks new feature “Cover Block uses Featured Image”

### ISSUE THREAD

**Issue by Galanos** [image-source-control/image-source-control#176]
This concerns the new feature available in **WP6** to use the Featured Image in a Cover block through a new and dedicated button on the block’s toolbar.

With ISC’s Overlay activated, the image isn’t shown and the Overlay is misplaced. I’m speculating that this is due to the semi-transparent black box placed above the image for text-image contrast. This bug seems to be independent of theme choices.

I couldn’t find proper documentation on any WordPress site but [this Youtube video](https://www.youtube.com/watch?v=2-1kwfDzZXg) shows off the feature.

How to reproduce
---

1. Activate the Overlay option **in WordPress 6 Beta**
2. Define a Featured Image for a post, use one with credit metadata
3. Insert a Cover block into the post
4. Activate the “Use featured image” option in the toolbar (see screenshot below) for the Cover block
5. See the frontend

<img width="671" alt="Bildschirmfoto 2022-05-17 um 11 21 49" src="https://user-images.githubusercontent.com/2743446/168777583-d211605f-fa7c-4768-ad03-18e6a82d3afb.png">

Expected behavior
---

- [ ] the Featured Image inside a cover block should display correctly and have its Overlay positioned and displayed correctly

**Screenshots**

<img width="774" alt="Bildschirmfoto 2022-05-17 um 11 51 08" src="https://user-images.githubusercontent.com/2743446/168783557-5fdbd62f-fde4-463e-bc9e-f510dbbb0318.png">

Priority
---

High priority for the WP6 release on May 24.

Additional context
---

There seems to be no relation to the original Featured Image placement, the image is handled identically to other Cover block pictures—it’s just the same source as the original Featured Image.

Tests
---

—

**Comment by webzunft** [comment]
The problem here is our CSS that gives the wrapping `.isc-source` container the rule `position: relative;`. Removing or overriding this rules seems to help:

`.wp-block-cover .isc-source { position: static; }`

**Comment by Galanos** [comment]
Ah, yes, that’s how I fixed an issue I had with the demo site I’m currently setting up: The theme offers to overlay the Featured Image of a post with a (tinted, semi-transparent) box—for contrast to the white heading that’s placed on top of the image. When I used that overlay, the image was pushed out of the wrapping box and I had to set the `isc-source` span to `position: initial` to get it back inside. Pretty much the same setup as with the new Cover block with Featured Image, now that I think of it.
